pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -36,7 +36,7 @@ func newEcNegateHint(point hinter.ResOperander) hinter.Hinter {
 				return err
 			}
 
-			pointMemoryValues, err := hinter.GetConsecutiveValues(vm, pointAddr, int16(6))
+			pointMemoryValues, err := mem.InitializeEmptyMemory().GetConsecutiveValues(pointAddr, int16(6))
```

Comment:
you need to do as @MaksymMalicki  pointed out `vm.Memory`

Reply:
Done 👍🏽